app/views/installation/onboarding/index.html.erb

```diff
@@ -43,9 +43,9 @@
                 </label>
                 <div class="update-subscription--checkbox">
                   <%= check_box_tag "subscribe_to_updates", 'true', true %>
-                  <div for="subscribe_to_updates">
-                  Subscribe to release notes, newsletters & product feedback surveys.
-                  </div>
+                  <label for="subscribe_to_updates">
```

Comment:
Can we use a class name here for this label and write target style for it to avoid the `!important`?